### DISCUSSION TITLE
Best gain setting airspy mini for VHF conventional analog NFM

### DISCUSSION THREAD

**Discussion by israelgoldstein** [rtl-airband/RTLSDR-Airband#404] [Ideas]
What's the gain settings you use ?
And maybe share your config file ?

**Comment by charlie-foxtrot** [comment]
Gain and offset are very specific to your hardware / installation / site / frequency.

I use SDR++ (https://www.sdrpp.org) with each dongle in "tcp stream mode" to visualize the spectrum and chose gain / offset.  It really is more of an art than anything else, and I'll end up with different settings for multiple dongles running off of the same splitter.
